nixos/modules/services/web-apps/filebrowser.nix

```diff
@@ -0,0 +1,181 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) generators types hasPrefix literalExpression mkEnableOption mkIf
+                mkOption mkPackageOption optional optionalAttrs;
+
+  cfg = config.services.filebrowser;
+  enableDynamicUser =
+    assert cfg.user != "filebrowser" -> cfg.group != "filebrowser";
+    cfg.user == "filebrowser";
```

Comment:
I find it kinda strange and unexpected that DynamicUser gets disabled when the user and group settings are changed.